examples/Update examples of fuzzy quantifiers.py

```diff
@@ -0,0 +1,122 @@
+from frlearn.uncategorised import fuzzy_quantifers as fuzz
+from frlearn.uncategorised import quantifiers
+import numpy as np
+
+print(
+    "We present some examples of the evaluation of unary and binary sentences using Zadeh's model, Yager's model and WOWA's model")
+print(
+    '***************************************************************************************************************************')
+print()
+
+# Definition of the RIM quantifiers some and most
+some = quantifiers.QuadraticSigmoid(0.1, 0.4)
+most = quantifiers.QuadraticSigmoid(0.3, 0.9)
+
+# Definition of the base set
+set_v = [i + 1 for i in range(10)]
+
+
+# Definition of fuzzy sets
+def A(x):
+    if x <= 2:
+        return 0
+    elif 2 < x <= 5:
+        return (x - 2) / 10
+    else:
+        return 1
+
+
+A_list = [A(x) for x in set_v]
+
+
+def B(x):
+    if x <= 4:
+        return (x) / 5
+    elif 4 < x <= 8:
+        return (x - 4) / 8
+    else:
+        return (x - 8) / 10
+
+
+B_list = [B(x) for x in set_v]
+
+
+# Definition of the Kleene-Dienes's implicator of A and B
+def Kleene_Dienes(x):
+    return max(1 - A(x), B(x))
+
+
+Kleene_Dienes_list = [Kleene_Dienes(x) for x in set_v]
+
+print(A_list, 'and ', B_list, 'are two fuzzy sets of the crisp set ', set_v)
+print()
+
+# Evaluation of unary sentences of the form “Λ X’s are A’s”, where Λ is a RIM quantifier,in Zadeh’s model and Yager's model
+Zadeh_unary_evaluation = fuzz.unary_zadeh_model(most, A_list)
+print("The evaluation of the unary sentence 'Most elements of", set_v, "are ", A_list, "' \n using Zadeh's model is ",Zadeh_unary_evaluation)
+print()
+
+
+# Yager's model
+# Definition of the values of the monotone measure in the unary Yager's model
+def mu_yager_unary(x):
+    return some((len(x) / len(set_v)))
```

Comment:
This feels like it should be part of the `unary_yager_model` function in the `fuzzy_quantifiers` module, with a parameter allowing the user to specify the quantifier in question (maybe with `some` as the default).